### PR TITLE
chore: cleanup experiment for search layout upgrade;

### PR DESCRIPTION
**Pull request by ilasw**
## Changes
- cleanup experiment for search layout upgrade;

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://mi-478-cleanup-search-page.preview.app.daily.dev

Search page: https://mi-478-cleanup-search-page.preview.app.daily.dev/search?q=react

### CODE REVIEW

File: packages/shared/src/components/widgets/WidgetCard.tsx [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 export const WidgetCard = (props: WidgetCardProps): ReactElement => {
-  const { heading, className, children } = props;
+  const { heading, className, children, ...attrs } = props;
   return (
     <section
       className={classNames(
         'flex flex-col rounded-16 border border-border-subtlest-tertiary',
         className,
       )}
+      {...attrs}
```
Comment (capJavert) [inline packages/shared/src/components/widgets/WidgetCard.tsx:17]: What is this for?

File: packages/shared/src/hooks/search/useSearchResultsLayout.ts [hunk 2]
```diff
@@ -1,33 +1,18 @@
...
-    },
-  );
-
-  const isSearchResultsUpgrade =
-    !isLoading && isSearchUpgradeExperiment && shouldShowNewLayout;
-
   return {
-    isSearchResultsUpgrade,
-    isLoading: !isTesting && shouldShowNewLayout && isLoading,
+    isSearchResultsUpgrade: shouldShowNewLayout,
```
Comment (capJavert) [inline packages/shared/src/hooks/search/useSearchResultsLayout.ts:15]: This should be always true now right if experiment won, so we don't need the hook at all potentially?

File: packages/shared/src/components/widgets/WidgetCard.tsx (hunk 1 above)
Comment (ilasw) [reply packages/shared/src/components/widgets/WidgetCard.tsx:17]: I'm using it to pass potential attributes that comes from `ComponentProps<'section'>`. 

I'm passing  `data-testid="related-tags"` in Related tags for eg.

File: packages/shared/src/hooks/search/useSearchResultsLayout.ts (hunk 2 above)
Comment (ilasw) [reply packages/shared/src/hooks/search/useSearchResultsLayout.ts:15]: We still have two check to do in multiple places: `!isLaptop && feedName === SharedFeedPage.Search`. 
I can remove it but have to replace with this logic in 4 files 🤔

**Review by capJavert — APPROVED** [review]

File: packages/shared/src/hooks/search/useSearchResultsLayout.ts (hunk 2 above)
Comment (nensidosari) [reply packages/shared/src/hooks/search/useSearchResultsLayout.ts:15]: Seems like in some pages where this is used, looks like only the isLaptop check is needed rather then checking what feed page we're in.

If we're keeping this though, I would name it isSearchPageLaptop.

File: packages/webapp/__tests__/SearchResultsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import React from 'react';
```
Comment (nensidosari) [inline packages/webapp/__tests__/SearchResultsPage.tsx:1]: Thank u

File: packages/shared/src/hooks/search/useSearchResultsLayout.ts (hunk 2 above)
Comment (nensidosari) [inline packages/shared/src/hooks/search/useSearchResultsLayout.ts:15]: If we're keeping this though, I would name it isSearchPageLaptop.

**Review by nensidosari — APPROVED** [review]
Just one minor thing, else looks good

File: packages/shared/src/hooks/search/useSearchResultsLayout.ts (hunk 2 above)
Comment (ilasw) [reply packages/shared/src/hooks/search/useSearchResultsLayout.ts:15]: Updated! ✅

**Review by nensidosari — APPROVED** [review]

**Pull request merged**